lib/url.c

```diff
@@ -1121,6 +1121,12 @@ ConnectionExists(struct Curl_easy *data,
         /* connect-only or to-be-closed connections will not be reused */
         continue;
 
+      if(extract_if_dead(check, data)) {
+        /* disconnect it */
+        (void)Curl_disconnect(data, check, /* dead_connection */TRUE);
```

Comment:
Why the cast?

Reply:
It was there already. We `(void)` cast at times when we want to emphasize that we know the function returns an error code but we willfully ignore that.

Reply:
Also some compilers and code tools may warn if a function result is ignored unless you explicitly (void) it.